yarf/rf_libraries/resources/kvm.resource

```diff
@@ -340,3 +340,35 @@ Ensure ${destination} Does Not Match
     IF    ${match}
         Fail                    Found ${destination} where it wasn't intended or expected
     END
+
+Drag ${target} And Drop On ${destination}
+    [Documentation]    Move the pointer to a target, drag and drop it on a destination.
+    ...
+    ...    Embedded arguments:
+    ...    - ${target}: Can be a string, or the path of an image template file representing
+    ...    the item to drag or a coordinate tuple (x, y) of integers representing the
+    ...    absolute position to drag from. For details please see `Move Pointer to ${destination}`.
+    ...    - ${destination}: Can be a string, or the path of an image template file representing
+    ...    the target location to drop on, or a coordinate tuple (x, y) of integers representing the
+    ...    absolute position to drop on. For details please see `Walk Pointer To ${destination}`.
+    ...
+    [Arguments]             ${step_distance}=16     ${delay}=0.01
+
+    Move Pointer To ${target}
+    ${end_position}=        Drag And Drop On ${destination}                 ${step_distance}        ${delay}
+    RETURN                  ${end_position}
+
+Drag And Drop On ${destination}
+    [Documentation]    Drag and drop the current target (where the pointer is) on a destination.
+    ...
+    ...    Embedded arguments:
+    ...    - ${destination}: Can be a string, or the path of an image template file representing
+    ...    the target location to drop on, or a coordinate tuple (x, y) of integers representing the
+    ...    absolute position to drop on. For details please see `Walk Pointer To ${destination}`.
+    ...
+    [Arguments]             ${step_distance}=16     ${delay}=0.01
+
+    Press LEFT Button
+    ${end_position}=        Walk Pointer To ${destination}                  ${step_distance}        ${delay}
```

Comment:
This makes the keyword compatible only with images, not text. I think you should recognize if ${destination} is a path or a string, get the coordinates, and then use `Walk Pointer To Absolute`.